### PR TITLE
Gallywix Hat

### PR DESCRIPTION
**Pull request by panpoleon**
# Changelog
- Added goblin top hat used by kings and emperors.

### CODE REVIEW

File: gfx/models/portraits/attachments/female_headgears/secular/goblin/gallywix_hat_01/female_headgear_secular_goblin_gallywix_hat_01.asset [hunk 1]
```diff
@@ -0,0 +1,19 @@
+﻿pdxmesh = {
```
Comment (zumbak04) [inline gfx/models/portraits/attachments/female_headgears/secular/goblin/gallywix_hat_01/female_headgear_secular_goblin_gallywix_hat_01.asset:1]: The female top hat clips with the head sometimes.
<details>
<summary>Click to expand</summary>

image

</details>

**Review by zumbak04 — CHANGES_REQUESTED** [review]

**Review by Grimlawck — DISMISSED** [review]

**Review by Grimlawck — CHANGES_REQUESTED** [review]
I went through the years 583-605 and the kings and Emperor had their top hats. However, there is one particular Duke who has a top hat as well even though he isn't a king. Duke Jazikrax of Booty Bay and this persists throughout all time periods.
<details>
<summary>Click to expand</summary>



</details>

In 603 it appears the characters don't have their top hats but after clicking on them their portraits corrected themselves.  (Notice the child before and after as well)
<details>
<summary>Before</summary>



</details>

<details>
<summary>After</summary>



</details>

In 605 all is normal besides the Duke spoken about previously so I wont bother with a picture.

**Comment by zumbak04** [comment]
> I went through the years 583-605 and the kings and Emperor had their top hats. However, there is one particular Duke who has a top hat as well even though he isn't a king. Duke Jazikrax of Booty Bay and this persists throughout all time periods.

Fixed in 68ef6e1f08f0eac91373488caeeab1509e7a6a5e.

> In 603 it appears the characters don't have their top hats but after clicking on them their portraits corrected themselves. (Notice the child before and after as well)

It's fine. The Character Finder may be outdated, it updates on clicking characters.

**Review by Grimlawck — APPROVED** [review]
Once again I went through the different time periods just to be thorough. With the new changes, I see that the spouses of the rulers don't have any top hats anymore and the Duke that was mentioned previously no longer has one either. So everything looks as though it is intended.

**Review by zumbak04 — APPROVED** [review]
I wonder if you can make coloring patterns. It would make hats x10 better. Anyway, the current one LGTM!

**Review by TheDrPibb1999 — APPROVED** [review]
A hat fit for a trade-prince!!!

File: common/culture/cultures/goblin.txt [hunk 2]
```diff
@@ -2,12 +2,11 @@
 	graphical_cultures = {
```
Comment (zumbak04) [inline common/culture/cultures/goblin.txt:2]: It's just black for me.
<details>
<summary>Click to expand</summary>

image

</details>

**Review by zumbak04 — CHANGES_REQUESTED** [review]

**Review by zumbak04 — CHANGES_REQUESTED** [review]

File: common/culture/cultures/goblin.txt (hunk 2 above)
[the remark this answers is not in the document]
Comment (Grimlawck) [reply common/culture/cultures/goblin.txt:2]: I agree that it is kind of hard to tell the difference between the two colors because of how dark they are.

**Review by Grimlawck — APPROVED** [review]
Unless the new hats I saw in the development chat were supposed to be in this pull everything LGTM! All seems to work between the different time periods as well. Female top hat for the female rulers looks great as well.

**Review by zumbak04 — APPROVED** [review]
LGTM!

**Pull request merged**